pkg/workload/schemachange/operation_generator.go

```diff
@@ -4808,6 +4832,22 @@ func isClusterVersionLessThan(
 	return clusterVersion.Less(targetVersion), nil
 }
 
+// isMixedVersionTest works similarly to isClusterVersionLessThan, but without
+// specifying a version. It returns true if the cluster version is not the
+// latest, indicating a mixed-version test.
+func isMixedVersionTest(ctx context.Context, tx pgx.Tx) (bool, error) {
```

Comment:
super nit: it could be called `inMixedVersionState`, since technically the whole test could be considered a "mixed version test"